core/templates/dev/head/services/AssetsBackendApiService.ts

```diff
@@ -46,21 +46,22 @@ oppia.factory('AssetsBackendApiService', [
     var ASSET_TYPE_IMAGE = 'image';
 
     var GCS_PREFIX = ('https://storage.googleapis.com/' +
-      GLOBALS.GCS_RESOURCE_BUCKET_NAME + '/exploration');
+      GLOBALS.GCS_RESOURCE_BUCKET_NAME);
     var AUDIO_DOWNLOAD_URL_TEMPLATE = (
       (DEV_MODE ? '/assetsdevhandler' : GCS_PREFIX) +
-      '/<exploration_id>/assets/audio/<filename>');
+      '/<entity_type>/<entity_id>/assets/audio/<filename>');
     var IMAGE_DOWNLOAD_URL_TEMPLATE = (
       (DEV_MODE ? '/assetsdevhandler' : GCS_PREFIX) +
-      '/<exploration_id>/assets/image/<filename>');
+      '/<entity_type>/<entity_id>/assets/image/<filename>');
 
     var AUDIO_UPLOAD_URL_TEMPLATE =
       '/createhandler/audioupload/<exploration_id>';
 
     // Map from asset filename to asset blob.
     var assetsCache = {};
     var _fetchFile = function(
-        explorationId, filename, assetType, successCallback, errorCallback) {
+        entityId, entityType, filename, assetType, successCallback,
```

Comment:
Swap entity id and entity type, here and elsewhere.

Reply:
Done